xbmc/addons/AddonRepos.cpp

```diff
@@ -393,6 +471,14 @@ bool CAddonRepos::FindDependency(const std::string& dependsId,
             "ADDONS: found dependency [{}] for install/update from repo [{}]. dependee is [{}]",
             dependencyToInstall->ID(), repoForDep->ID(), parent->ID());
 
+  if (dependencyToInstall->HasType(ADDON_REPOSITORY))
+  {
+    CLog::Log(LOGDEBUG,
+            "ADDONS: ... has type ADDON_REPOSITORY and will not install!");
```

Comment:
Think here you have something missing and planned to add something on `...` after the Draft is done.